the-9-exercises/src/components/Fetch.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react";
+import UserDisplayComponent from "./UserDisplayComponent";
```

Comment:
Suggestion: I think there is no need to describe everything in the component naming something like 'User' would be enough

Reply:
I would say the same thing!